ardupilot_gz_bringup/package.xml

```diff
@@ -16,9 +16,9 @@
   <exec_depend>ardupilot_sitl_models</exec_depend>
   <exec_depend>launch</exec_depend>
   <exec_depend>launch_ros</exec_depend>
-  <exec_depend>ros_gz_bridge</exec_depend>
   <exec_depend>ros_gz_sim</exec_depend>
-  <exec_depend>ros_gz_bridge</exec_depend>
+  <!-- TODO this installs ignition -->
```

Comment:
Do we want to keep this if `rosdep` installs ignition? It may be better to refer users to the Gazebo Sim documentation for running non-standard ROS 2 / Gazebo combos: https://gazebosim.org/docs/latest/ros_installation.

Reply:
I was not sure. If there were other rosdep keys, I would say yes, but I haven't found them. We still want to ensure the build order is correct. Perhaps I should ask on the discord server.

Reply:
I didn't get an answer. Let's leave it for now; I don't think it will break anything.